src/lib.rs

```diff
@@ -228,39 +236,49 @@ async fn save<R: Runtime>(
   with_store(&app, stores, path, |store| store.save(&app))
 }
 
-pub struct StorePlugin<R: Runtime> {
-  invoke_handler: Box<dyn Fn(Invoke<R>) + Send + Sync>,
-  stores: Option<HashMap<PathBuf, Store>>,
+#[derive(Default)]
+pub struct PluginBuilder {
```

Comment:
I found `StorePluginBuilder` to be pretty unwieldy and I think it's fine to let users import the Builder under a different name if needed

Reply:
makes sense to me